Scyther Playground/Scyther Playground/Button.swift

```diff
@@ -170,12 +170,17 @@ class Button: UIButton {
         super.init(frame: frame)
 
         //Update Constraints
-        snp.makeConstraints { (make) in
-            make.height.equalTo(type.buttonHeight)
-        }
+        heightAnchor.constraint(equalToConstant: type.buttonHeight).isActive = true
     }
 
     private func setType(_ type: ButtonStyle) {
+        guard let titleLabel = titleLabel,
+              let imageView = imageView,
+              let superview = superview
+        else {
+            return
+        }
+        
```

Comment:
Again, preference is to not unwrap unnecessarily. Nothing in this file requires that these are explicitly not null.

Reply:
same as above i could solve it like this if preferred:

```
        if let textLabel = textLabel {
            NSLayoutConstraint.activate([
                textLabel.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 16),
                textLabel.leftAnchor.constraint(equalTo: contentView.leftAnchor, constant: 16),
                textLabel.rightAnchor.constraint(equalTo: contentView.rightAnchor, constant: -16)
            ])
        }
```